lib/Dialect/FIRRTL/Transforms/EmitOMIR.cpp

```diff
@@ -799,6 +789,7 @@ void EmitOMIRPass::emitTrackedTarget(DictionaryAttr node,
         LLVM_DEBUG(llvm::dbgs() << "Marking NLA-participating instance "
                                 << innerRef.getName() << " in module "
                                 << modName << " as dont-touch\n");
+        tempSymInstances.erase(instOp);
```

Comment:
If you don't mind, what's the reason for adding this line (and similar line below, L815)?

Reply:
`EmitOMIR` is adding temporary symbols to `InstanceOp` for creating temporary `HierPathOp`, and then removing the symbols and the `HierPathOp`. Earlier this was done in the `getOrAddInnerSym`. So I have copied this to wherever the function was getting called. In a further cleanup, I think we can get rid of the temporary `HierPathOp` or even leave the symbols on the `InstanceOp`. I didn't want to change any behavior in this PR.
```
StringAttr EmitOMIRPass::getOrAddInnerSym(Operation *op) {
  tempSymInstances.erase(op);
```

Reply:
Great, thanks!